build.go

```diff
@@ -37,7 +37,7 @@ import (
 	"github.com/buildpacks/pack/internal/style"
 	"github.com/buildpacks/pack/logging"
 	"github.com/buildpacks/pack/pkg/archive"
-	"github.com/buildpacks/pack/project"
+	projectCommon "github.com/buildpacks/pack/project/common"
```

Comment:
I wonder if there's a better name for this 🤷

Reply:
No strong feeling over here!